index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" >
+  <title>Document</title>
+</head>
+<body>
+  <article>
+    <p>This is a pragraph</p>
```

Comment:
It might be a good idea to give your element a class attribute so you can give them easily an style.